server/chat-plugins/uno.ts

```diff
@@ -810,7 +810,7 @@ export const commands: ChatCommands = {
 		},
 
 		color(target, room, user) {
-			const game = room.game as UnoGame;
+			const game = room.getGame<UnoGame>(UnoGame);
 			if (!game || game.gameid !== 'uno') return false;
```

Comment:
don't need the `gameid` checks anymore, same with a few more places in this file.